frontend/src/pages/enabledApplications/EnabledApplications.tsx

```diff
@@ -10,31 +12,54 @@ import './EnabledApplications.scss';
 const description = `Launch your enabled applications or get started with quick start instructions
  and tasks.`;
 
+type EnabledApplicationsInnerProps = {
+  loaded: boolean;
+  loadError?: Error;
+  components: ODHApp[];
+};
+const EnabledApplicationsInner: React.FC<EnabledApplicationsInnerProps> = React.memo(
```

Comment:
question: similar question to before around checking... dual memos vs a single memo, specifically for the `components` list vs `EnabledApplicationsInner`
 
optional: understandable inner outer components, future break out, multiple